test/fixtures/minio-fixture/src/main/java/org/elasticsearch/test/fixtures/minio/MinioTestContainer.java

```diff
@@ -19,6 +19,14 @@ public final class MinioTestContainer extends DockerEnvironmentAwareTestContaine
     public static final String DOCKER_BASE_IMAGE = "minio/minio:RELEASE.2024-12-18T13-15-44Z";
     private final boolean enabled;
 
+    /**
+     * for packer caching only
+     * see CacheCacheableTestFixtures.
+     * */
+    protected MinioTestContainer() {
```

Comment:
fixes another packer cache issue when caching docker images we rely on in our fixtures: https://buildkite.com/elastic/elasticsearch-vm-images/builds/671#0195c53c-d566-405c-9503-afc95a6e6706